site/en/codelabs/openthread-border-router-nat64/index.lab.md

```diff
@@ -113,6 +113,10 @@ The NAT64 prefix will be used by Thread devices when communicating with an IPv4
 > 
 > **Note:** You may need to ensure IPv4 packet forwarding is enabled in  the system config. The content of `/proc/sys/net/ipv4/conf/default/forwarding` should be `1`.
 
+> aside negative
+> 
+> **Note:** OpenThread does not support the NAT64 well-known prefix `64:ff9b::/96`. Instead, Thread Border Routers dynamically generate their own NAT64 prefixes (as seen above) and publish them as necessary in the Thread Network Data. Thread End Devices must obtain this NAT64 prefix from the Thread Network Data and synthesize their own IPv6 addresses, see [`otNat64SynthesizeIp6Address()`](https://openthread.io/reference/group/api-nat64#otnat64synthesizeip6address). This is automatically done in the CLI, but needs to be specifically implemented in custom applications.
```

Comment:
Would suggest:

"OpenThread does not use the NAT64 well-known prefix `64:ff9b::/96`. Instead, Thread Border Routers dynamically published NAT64 prefixes used by the NAT64 translator (as seen above) in the Thread Network Data."

When AIL NAT64 is fully supported in the future, 64:ff9b::/96 could be a possible NAT64 prefix (provided by the ISP) in the network data, and it is not always "generated".